code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -198,3 +198,36 @@
 		locked = FALSE
 		icon_state = icon_broken
 		origin_tech = null //wipe out any origin tech if it's unlocked in any way so you can't double-dip tech levels at R&D.
+
+/obj/item/storage/lockbox/experimental_weapon
+	name = "class A-113 classified lockbox"
```

Comment:
```suggestion
	name = "A-113 classified lockbox"
```
I would just cut down on wordiness here, the presence of a numerical identifier implies some sort of classification or product identifier